Automattic/jetpack
# Admin Page: Remove prop-types related warnings from development builds

#### Changes proposed in this Pull Request:

* Applied the prop-types codemod
* Updates react-router (and `history` and `react-redux-router`) to use a version that relies on `prop-types`. 

#### Testing instructions:

* Check this branch
* Build the admin page
* Visit the Jetpack Dashboard and Settings pages.
* Expect to see no warnings like these:
   ```
  Warning: Accessing PropTypes via the main React package is deprecated, and will be removed in  React v16.0. Use the latest available v15.* prop-types package from npm instead.
   ```
* Switch around tabs and Jetpack Admin Pages to confirm the routes work OK.


#### Proposed changelog entry for your changes:
